core/modules/layout/layout.admin.inc

```diff
@@ -1468,6 +1468,7 @@ function layout_title_settings_form($form, &$form_state, Layout $layout) {
   $block_options = array();
   foreach ($layout->content as $uuid => $block) {
     $title = $all_blocks[$block->module][$block->delta]['info'];
+    if (isset($block->childDelta)) $title .= " - " . $block->childDelta;
```

Comment:
Although this one-liner is correct, I'd suggest to use curly braces, newlines and proper indent. Just for better code readability.